packages/components/src/overlay/src/Overlay.css

```diff
@@ -25,15 +30,14 @@
 .o-ui-overlay-arrow::before {
     content: "";
     transform: rotate(45deg);
-    background-color: var(--o-ui-bg-alias-surface);
-    box-shadow: var(--o-ui-bs-alias-floating);
+    background-color: transparent;
 }
 
 .o-ui-overlay-arrow::after {
     content: "";
     width: 20px;
     height: 10px;
-    background-color: var(--o-ui-bg-alias-surface);
+    background-color: transparent;
```

Comment:
for the :before and :after of arrow, I'd just put content: none and display: none; and remove other properties if we don't need it. Don't even really need display: none if you have content: none; but i like to put it anyway, just in case.

Reply:
At that point no need to declare :after

Reply:
true, I wasn't sure if maybe this was declared somewhere else so you had to override it.